src/preamble.js

```diff
@@ -794,6 +794,29 @@ var cyberDWARFFile = '{{{ BUNDLED_CD_DEBUG_FILE }}}';
 
 #include "URIUtils.js"
 
+#if ASSERTIONS
+function createExportWrapper(name, fixedasm) {
+  return function() {
+    var displayName = name;
+#if !WASM_BACKEND
+    if (name[0] == '_') {
+      displayName = name.substr(1);
+    }
+#endif
+    var asm = fixedasm;
```

Comment:
the param could be `asm` and then these 4 lines would be `asm = asm || Module['asm'];`

Reply:
I found that I needed a new var binding.  In that case where this function is called multple times I need to remember the original closed over argument `fixedasm`.   If I just assign to that argument on the fist call I loose access to the original closed over binding (I think).

Imagine that the first time the function is called `Module['asm']` is a valid object that doesn't have the right keys in yet.   Then the second time its called `Module['asm']`  is the real instance.   We can to potentially loop up `Module['asm']`  on each call.    This seems to happen in in the `browser.test_runtime_misuse` test.

Reply:
Ah yes, you're right. It is subtle here. `Module.asm` can change over calls to the create method, so we need to get the new value every time. So this can't be simplified.